hydrabus_framework/utils/generic.py

```diff
@@ -0,0 +1,75 @@
+import time
```

Comment:
You should find another name to your file. Generic is not explicit

Reply:
Done [#b2d6a98](https://github.com/hydrabus-framework/framework/commit/b2d6a9836a7edc9b167ac832963c04e0c4cca333)